app/models/openid_user_identity.rb

```diff
@@ -16,7 +16,7 @@ class OpenidUserIdentity < ::UserIdentity
   # @return [OpenidUserIdentity]
   def self.build_from_okta_profile(uuid:, profile:, ttl:)
     identity = new(
-      uuid: profile['uuid'] || uuid,
+      uuid: uuid,
```

Comment:
The code here was reverted but the comment on line 13 still references the fallback behavior.

Reply:
Removed!